six/modules/c++/six.sidd/include/six/sidd/SFA.h

```diff
@@ -88,8 +88,9 @@ struct SFAPoint : public SFAGeometry
     {
```

Comment:
As long as we're touching this code, update these constructors to initialize these members via initializer list instead of in the constructor body